pkg/cmd/roachtest/tests/decommissionbench.go

```diff
@@ -675,6 +675,15 @@ func runDecommissionBench(
 		bytesUsedMetric,
 	)
 
+	defer func() {
+		if err := exporter.Close(func() error {
+			uploadPerfArtifacts(ctx, t, c, benchSpec, pinnedNode, workloadNode, perfBuf)
```

Comment:
Are we uploading it twice? L506 uploads it to `pinnedNode`.

Reply:
It uploads only once, since we go through `uploadPerfArtifacts`. `exporter.Close` doesn't upload the file.

Reply:
Thanks, I was confused by the truncated diff. I think GH UI could benefit from placing a function name above the truncated code block.